assets/js/account/b4/account-order-detail.tpl.liquid

```diff
@@ -0,0 +1,143 @@
+                                            <span>{{ 'customer.order.placed_on_date_html' | t: created_date }}</span>
+                                        </div>
+                                        <div ng-if="$ctrl.order.isCancelled">
+                                            {% capture cancelled_on %}<span ng-bind="$ctrl.order.modifiedDate | date: 'medium'" class="font-weight-normal"></span>{% endcapture %}
+                                            <div>
+                                                {{ 'customer.order.cancelled_on' | t: cancelled_on }}
+                                            </div>
+                                            <div>{{ 'customer.order.cancel_reason' | t }}: {% raw %}{{ $ctrl.order.cancelReason }}{% endraw %}</div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="py-0" ng-if="$ctrl.order">
+                     <div class="row">
+                        <div class="col">
+                            <vc-line-items order="$ctrl.order"></vc-line-items>
+                        </div>
+                     </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="col-md-3">
+        <div class="row">
+            <div class="col">
+                <div class="row mb-3">
+                    <div class="col-6 pl-0 pr-2">
+                        <button class="btn btn-primary btn-lg btn-block" ng-click="$ctrl.getInvoicePdf()">
+                            <span>{{ 'customer.order.get_invoice' | t }}</span>
+                        </button>
+                    </div>
+                    <div class="col-6 pl-2 pr-0">
+                        <button class="btn btn-primary btn-lg btn-block">
+                            <span>{{ 'customer.order.reorder' | t }}</span>
+                        </button>
+                    </div>
+                </div>
+                <div class="row mt-3">
+                    <div class="col p-0">
+                        <div class="card">
+                            <div class="card-header text-uppercase text-condensed font-weight-bold">Order totals</div>
+                            <div class="card-body">
+                                <vc-totals order="$ctrl.order"></vc-totals>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="row mt-3" ng-if="$ctrl.deliveryMethod.type =='shipping'">
+                    <div class="col p-0">
+                        <div class="card">
+                            <div class="card-header text-uppercase text-condensed font-weight-bold">{{ 'customer.order.billing_address' | t }}</div>
```

Comment:
The result will be exactly the same, if you will use `<h5 class="card-header text-uppercase">`

Reply:
Probably even without "text-uppercase", since it's <h5>

Reply:
@ilyawzrd For some reason h5 doesn't have uppercase style. May be we may fix it, but I'm not sure it doesn't used anywhere